tests/test_serializer.py

```diff
@@ -87,6 +90,25 @@ def validate(self, attrs):
         assert not serializer.is_valid()
         assert serializer.errors == {'char': ['Field error']}
 
+    def test_serializer_with_validator(self):
+        class ExampleModel(models.Model):
+            username = models.CharField(max_length=100)
+
+        class ExampleSerializer(serializers.ModelSerializer):
+            class Meta:
+                model = ExampleModel
+                fields = '__all__'
+                extra_kwargs = {
+                    'username': {
+                        'validators': [
+                            RegexValidator('^\w+$'),
+                        ],
+                    },
+                }
```

Comment:
Let's reduce this case to a regular `Serializer`, rather than a `ModelSerializer`.